code/game/objects/items/stacks/stack.dm

```diff
@@ -339,7 +339,7 @@
 	return ..()
 
 /obj/item/stack/restock()
-	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical))
+	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical) || istype(src,/obj/item/stack/package_wrap))
```

Comment:
No, he means you can have this:
`/obj/item/stack/restock()
   return
/obj/item/stack/cable_coil()
   //Code goes here`

And then add that to each of the three. Or if you want to be really tidy you can repath all three into subtypes of something like stack/restockable/* but that would require map editing so...